infrastructure/server-setup/tasks/checks.yml

```diff
@@ -14,5 +14,4 @@
       - mongodb_admin_username is defined
       - mongodb_admin_password is defined
       - elasticsearch_superuser_password is defined
-      - disk_encryption_key is defined
```

Comment:
Disk encryption made optional as it's quite difficult to get working on local VMs